firmware/validators.py

```diff
@@ -0,0 +1,96 @@
+"""Validators - functions to validate stuff"""
+
+from firmware.repository import get_company, get_username_by_id
+
+
+def validate_user_type(current_session, company_id):
```

Comment:
Please use this http://flask.pocoo.org/docs/0.12/patterns/wtforms/ for all the forms (and validations) you have in this project :)